hoot-rnd/src/main/cpp/hoot/rnd/io/MultiaryIngester.h

```diff
@@ -49,6 +49,16 @@ class ElementInputStream;
  *
  * This class requires that the input be a streamable format, the output layer be a Hootenanny
  * API database layer, and the changeset output format be a Spark changeset.
+ *
+ * TODO: Use of OsmFileSorter by this class can be eliminated by swapping it for
+ * ExternalMergeElementSorter.  See #2622
+ *
+ * TODO: 9/12/18 - I believe that there is a critical bug in that hoot api db data sources aren't
+ * be sorted before changeset derivation.  It was originally believed all results were coming back
+ * completely sorted by ID from the db, but I don't believe that's the case (see
+ * ApiDb::selectElements).  It may not be a hard fix to hook the db data source up to
+ * ExternalMergeElementSorter or make the results come back sorted by ID, but given this code is
+ * still part of unused prototype, I'm not spending time on it right now - BDW
  */
```

Comment:
#2622 will handle getting rid of OsmFileSorter completely which removes all calls to osmosis and also removes an external call to Unix sort.